src/components/NurseryWithdrawals/PlantingProgressMapDialog.tsx

```diff
@@ -44,6 +46,12 @@ export default function PlantingProgressMapDialog({
     return Object.values(species).reduce((prev, curr) => prev + curr, 0);
   }, [species]);
 
+  useEffect(() => {
```

Comment:
i think we can remove this.. we can find the zone id at the time of deciding whether to open the modal.. and pass that to the modal.. which can pass it back onSubmit